rules.module

```diff
@@ -98,3 +99,45 @@ function rules_entity_update(EntityInterface $entity) {
     $event_dispatcher->dispatch("rules_entity_update:$entity_type_id", $event);
   }
 }
+
+/**
+ * Implements hook_help().
+ */
+function rules_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    case 'help.page.rules':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('Rules documentation is kept online. Please use the links below for more information about Rules. Feel free to contribute to improving the online documentation! For more information, see the <a href="!action" target="_blank">online documentation for the Rules module.</a>',
+              array('!action' => 'https://www.drupal.org/project/rules')) . '</p>';
```

Comment:
please use the short array syntax. And the "!" placeholder should not be used anymore, see https://www.drupal.org/node/2506427